torch/csrc/jit/mobile/flatbuffer_loader.cpp

```diff
@@ -93,7 +92,7 @@ class FlatbufferLoader final {
 
   typedef TypePtr (*TypeResolver)(
```

Comment:
Interesting use of typedef could "using" be used more cleanly here?

Reply:
Applied